src/dependencies/bootstrap/Gemfile

```diff
@@ -0,0 +1,12 @@
+# Ruby Gems for building and testing Bootstrap
```

Comment:
What dependencies dir is doing? We will not run any bootstrap script, instead we use RPMs to install packages. If you need list of packages we'll need to create spec files for them and then build their RPMs. Please explain how you install stuff and how dependencies dir is working with its internals. This should be solved before I'll make a build.